torch/fx/graph_module.py

```diff
@@ -271,6 +271,17 @@ def _get_attr(model: torch.nn.Module, attr_name: str):
     return _get_attr_via_attr_list(model, attr_name.split("."))
 
 
+def _del_attr(model: torch.nn.Module, attr_name: str):
```

Comment:
Could you not use `_get_attr_via_attr_list` on prefix to get the final `t`? That might be better for code reuse.

Reply:
Yep sounds good!